content/gaad-2022.md

```diff
@@ -0,0 +1,106 @@
+
+Of course, there is a forever-long list of accessibility improvements to be made across the web; to avoid trying to boil the proverbial ocean, it was important to set a north-star goal to help orient all efforts. 
+
+To that end, the goal became this: <strong>no <em>new</em> Ember app should fail accessibilty conformance checks.</strong> By this, we mean running the `ember new MY-APP` command, and adding at least one additional route (`ember generate route NAME`).
+
+## Improvements
+
+There have been some distinct areas of improvement, and I want to take the time to highlight and celebrate them!
+
+### Community Education
+
+One of the ways our community shares big ideas is through [EmberConf](https://emberconf.com), our primary annual conference. Thanks to the specific efforts of the conference program committee and the speakers who submitted CFPs, we witnessed a shift: talks began including more accessibility. In the past two Ember conferences, nearly every talk has directly discussed accessibility as it relates to the subject matter being presented. From "nearly none" to "almost all" over the space of 2-3 years is really something special; kudos to the conference speakers for providing this valuable information to our community!
+
+This year, I <em>especially</em> loved [Space Jamming Accessibility by Zoë Bijl & Jenny Judova](https://www.youtube.com/watch?v=QJxtGuJ2E0U). Want to browse all of talks and learn more? Check out our YouTube channel: https://www.youtube.com/c/EmberMeetupVideos
+
+Additionally, the #talk-a11y channel on the [Ember Community Discord Server](https://discord.gg/emberjs) has been a great place for folks to ask questions and get help implementing accessible interfaces in their Ember apps. The top question has probably been, "how do I make an accessible card component?" and we definitely still have [Inclusive Components](https://inclusive-components.design/cards/) to thank for providing <em>such</em> a great solution that it's permanently pinned to that channel as an answer!
+
+## Framework Improvements
+
+### Page Titles
+
+The first framework improvement to land was adding page titles by default. When you generate a new route in Ember, the corresponding page template will have `{{page-title "ROUTE NAME"}}` at the very top of the template. Users only have to provide a meaningful page name and the framework will put it in the correct place! 
+
+### Lang Attribute
+`--lang`, //TODO 
+
+### Unique Id Support
+
+In Ember v.4.4.0, the `{{unique-id}}` helper will be included by default in new Ember apps. Developers can use this helper to generate a unique ID string suitable for use as an ID attribute in the DOM. Notably, this removes the need for a backing class since it can be directly used in template-only components!
+
+```handlebars
+<input id={{unique-id}} type="email" />
```

Comment:
I wonder if maybe we should show the id passed into a component, like 

```suggestion
<MyFormRow @id={{unique-id}} />
```

And then show the associated label + input? It breaks the simplicity of the original example, but I think otherwise someone might use the helper directly on both their input and label, and assume that’s fine